### ISSUE TITLE
Help Getting Started

### ISSUE THREAD

**Issue by liorsbg** [mesos/storm#173]
Hi,
I'm very new to all of this. I just got a cluster up and running with DCOS and Marathon.
I'm a bit confused as to where to get started. Should I run the README instruction on the dcos-master?
I tried creating a Marathon app with the [instruction](https://github.com/mesos/storm#running-storm-on-marathon) in the README, but I'm definitely missing something.

Thanks!

**Comment by erikdw** [comment]
hey @liorsbg.  I don't use DCOS or Marathon, so cannot speak to how to run within those systems.  The existing documentation in our project was written by @brndnmtthws, maybe he can help if the instructions you linked are insufficient.

The way I and my company run this project is by launching the MesosNimbus daemon outside of the Mesos Cluster.  We use [runit](http://smarden.org/runit/) to launch & supervise the daemon, but there are a myriad of other options for running daemons.

**Comment by dinoba** [comment]
Is there any way to update documentatio, how to start Storm over Marathon?

**Comment by DarinJ** [comment]
I've done this and could write something up over the weekend.

**Comment by dinoba** [comment]
Looking forward. Thanks

**Comment by brndnmtthws** [comment]
There is actually an example in the readme already: https://github.com/mesos/storm#running-storm-on-marathon

Perhaps it could be updated with more detail, however.

**Comment by dinoba** [comment]
I think it should be updated. 
Not all of us are experts, so documentation should be more user friendly. Currently I am just investigating this as possible solution for our product. And  mesos, docker, marathon is not something I use everyday.
,

**Comment by liorsbg** [comment]
I agree, it would be super helpful if the docs were spiffed up just a bit. @erikdw @brndnmtthws those are the docs I linked to in the original question when I opened this issue, unfortunately I was not able to make them work.

**Comment by erikdw** [comment]
@JessicaLHartog : as part of updating the docs for marathon, we need to account for the issues mentioned in #174 and #175.
